src/Extensions/AzureBlobStorage/EgressArtifactSettings.cs

```diff
@@ -6,11 +6,6 @@ namespace Microsoft.Diagnostics.Monitoring.AzureBlobStorage
 {
     internal sealed class EgressArtifactSettings
     {
-        /// <summary>
-        /// The content encoding of the blob to be created.
-        /// </summary>
-        public string ContentEncoding { get; set; }
```

Comment:
I noticed last week that this is never set - when we were assigning `headers.ContentEncoding = artifactSettings.ContentEncoding`, this never has a value. Since this is something we're not currently doing, the idea was to take it out so that it's not part of the extension contract for `EgressArtifactSettings` if it's empty; however, if we end up using it, then it could be integrated again. Alternatively, we can leave this in if people prefer - I just wanted to call attention to the fact that it's not currently used.

Reply:
If it's not used, then removing it fine. We can always add it back later.